src/shared/helpers.ts

```diff
@@ -67,6 +67,15 @@ export const getWalletInformation = async (
   return json;
 };
 
+export const getJWKS = async (walletAddressUrl: string) => {
+  type JWK = { kid: string; alg: string; x: string; crv: string; kty: string };
+  type JWKS = { keys: JWK[] };
```

Comment:
I think we might have these types in `@interledger/open-payments`. Can you double check?